site/assets/js/snippets.js

```diff
@@ -60,14 +60,14 @@
       toast.show()
     })
 
-  // Instantiate all toasts in a docs page only
+  // Instantiate all toasts in docs pages only
   const toastTrigger = document.getElementById('liveToastBtn')
   const toastLiveExample = document.getElementById('liveToast')
+  const toastBootstrap = new bootstrap.Toast(toastLiveExample)
+
   if (toastTrigger) {
     toastTrigger.addEventListener('click', () => {
-      const toast = new bootstrap.Toast(toastLiveExample)
```

Comment:
I don't think it would make any difference, if we just change this to 
`bootstrap.Toast.getOrCreateInstace`, ensuring we handle it after clicking